.github/workflows/cypress.yml

```diff
@@ -6,7 +6,9 @@ jobs:
     name: Test
     runs-on: ubuntu-18.04
     strategy:
+      fail-fast: false
       matrix:
+        containers: [1, 2]
         browsers: [chrome, firefox]
         node-version: [14.x, 12.x]
```

Comment:
```suggestion
        node-version: [14.x]
```

You can drop this.

Reply:
Can you explain what you are suggesting here?

Reply:
It's just that we no longer support node 12, so there's no need to test for it.